.development

```diff
@@ -0,0 +1 @@
+ 
```

Comment:
What is the reason for adding this file?

Reply:
It allows the library to be edited without complaint within the ordinary Arduino IDE - not 100% convninced
 myself whether or not it is a good idea to have it in as standard for source code repos or not - it either needs to be tracked or in .gitignore.

Reply:
I often add a .development file to my own libraries for this reason, because I prefer to use the Arduino IDE for development work on the example sketches. However, I have it excluded from the repo.

There are two problems this file can cause:
- Libraries which contain this file are prohibited from being added to the Arduino Library Manager index: https://github.com/arduino/Arduino/wiki/Library-Manager-FAQ#how-is-the-library-list-generated
- It allows users to inadvertently modify the example sketch, causing confusion because they are accustomed to the library examples being immutable, known-good programs.

Reply:
Foudn  guidance here: [https://arduino.github.io/arduino-cli/latest/library-specification/#development-flag-file](https://arduino.github.io/arduino-cli/latest/library-specification/#development-flag-file) that says not to push - so I will delete and add it to .gitignore.

Reply:
I have made this change